src/intersection-observer-polyfill.js

```diff
@@ -244,8 +244,6 @@ export class IntersectionObserverPolyfill {
     // If opt_iframe is not provided, all LayoutRect has position relative to
     // the host document.
     elementRect = this.element_.getLayoutBox();
-    dev().assert(elementRect.width >= 0 && elementRect.height >= 0,
```

Comment:
😄 Didn't delete because the other one we accept elementRect as an input. Has no control of the input. However since the only usecase now lives inside `custom-element.js` I think it's fine to delete at this time.